test/components/views/rooms/DecryptionFailureBar-test.tsx

```diff
@@ -189,40 +190,12 @@ describe("<DecryptionFailureBar />", () => {
         act(() => {
             jest.advanceTimersByTime(5000);
         });
-        expect(getBar(bar)).toMatchSnapshot();
+        expect(getBar(bar)).toBe(null);
 
         bar.unmount();
     });
 
-    it("Recommends opening other devices if there are other verified devices", async () => {
-        ourDevice = verifiedDevice1;
-        allDevices = [verifiedDevice1, verifiedDevice2];
-        keyBackup = false;
-
-        const bar = render(
-            // @ts-ignore
-            <MatrixClientContext.Provider value={mockClient}>
-                <DecryptionFailureBar
-                    failures={[
-                        // @ts-ignore
-                        mockEvent1,
-                    ]}
-                />
-                ,
-            </MatrixClientContext.Provider>,
-        );
-
-        await waitFor(() => expect(mockClient.isSecretStored).toHaveBeenCalled());
-
-        act(() => {
-            jest.advanceTimersByTime(5000);
-        });
-        expect(getBar(bar)).toMatchSnapshot();
-
-        bar.unmount();
-    });
-
-    it("Displays a general error message if there are no other verified devices", async () => {
+    it("The banner is hidden if there are no other verified devices", async () => {
```

Comment:
```suggestion
    it("The banner is hidden if there is a key backup but no other verified devices", async () => {
```

Reply:
it seems like the code doesn't actually care about key backup or other verified devices?